nilearn/decoding/_utils.py

```diff
@@ -167,6 +176,19 @@ def check_feature_screening(
     """
     f_test = f_classif if is_classification else f_regression
 
+    if screening_percentile is None and screening_n_features is not None:
+        if mask_img is not None:
+            from nilearn.image import get_data
+
+            n_features_in_mask = np.sum(get_data(mask_img) != 0)
```

Comment:
Could it be sufficient to use `_get_data` instead of `get_data` ?

If not is it possible to move import to the top of the file?